annotation/tests/test_annotations.py

```diff
@@ -482,3 +484,34 @@ def test_construct_annotated_doc_db_error(
             )
             assert db.commit.assert_called_once()
             assert db.rollback.assert_called_once()
+
+
+def test_upload_json_to_minio():
+    test_json = json.dumps({"test": 1})
+    path_to_object = "path"
+    bucket_name = TEST_TENANT
+
+    with patch(
+        "annotation.annotations.main.bd_storage.get_storage"
+    ) as mock_get_storage:
+        mock_get_storage.return_value.upload_obj = Mock()
+
+        upload_json_to_minio(test_json, path_to_object, bucket_name, None)
```

Comment:
s3_resource is not optional. Pass mock if you dont need it

Reply:
The function doesn't utilize s3_resource anymore (because of bd_storage). Will pass in Mock()